spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/AbstractBinder.java

```diff
@@ -64,6 +66,12 @@
 
 	private volatile EvaluationContext evaluationContext;
 
+	protected final BinderErrorConfigurer errorConfigurer;
+
+	protected AbstractBinder(BinderErrorConfigurer errorConfigurer) {
+		this.errorConfigurer = errorConfigurer;
```

Comment:
I don't think we should, since we want this to be mandatory. This is actually the whole reason of this PR, making sure all binders adhere to a ErrorConfigurer strategy